splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SpliceUDTIT.java

```diff
@@ -57,159 +57,147 @@
+        }
     }
 
     @Test
     public void testUDA() throws Exception {
-        ResultSet rs = methodWatcher.executeQuery("select median(i) from t");
-        Assert.assertTrue(rs.next());
-        Assert.assertEquals(3, rs.getInt(1));
+        try(ResultSet rs = methodWatcher.executeQuery("select median(i) from t")) {
+            Assert.assertTrue(rs.next());
+            Assert.assertEquals(3, rs.getInt(1));
+        }
     }
 
     @Test
     public void testUDT() throws Exception {
-        ResultSet rs = methodWatcher.executeQuery("select orderID, customerID, totalPrice from orders");
-        Assert.assertTrue(rs.next());
-        int oid = rs.getInt(1);
-        int cid = rs.getInt(2);
-        Price price = (Price)rs.getObject(3);
-        Assert.assertEquals(12345, oid);
-        Assert.assertEquals(12, cid);
-        Assert.assertEquals(12, price.amount, 0.01);
-        Assert.assertTrue(price.currencyCode.compareTo("USD") == 0);
+        try(ResultSet rs = methodWatcher.executeQuery("select orderID, customerID, totalPrice from orders")) {
+            Assert.assertTrue(rs.next());
+            int oid = rs.getInt(1);
+            int cid = rs.getInt(2);
+            Price price = (Price) rs.getObject(3);
+            Assert.assertEquals(12345, oid);
+            Assert.assertEquals(12, cid);
+            Assert.assertEquals(12, price.amount, 0.01);
+            Assert.assertTrue(price.currencyCode.compareTo("USD") == 0);
+        }
     }
 
     @Test
-    public void TestSelectStatistics() throws Exception {
+    public void testSelectStatistics() throws Exception {
         methodWatcher.execute("analyze schema " + CLASS_NAME);
-        ResultSet rs = methodWatcher.executeQuery("select count(*) from sysvw.syscolumnstatistics");
-        Assert.assertTrue(rs.next());
-        Assert.assertTrue(rs.getInt(1)>0);
+        try(ResultSet rs = methodWatcher.executeQuery("select count(*) from sysvw.syscolumnstatistics") ) {
+            Assert.assertTrue(rs.next());
+            Assert.assertTrue(rs.getInt(1) > 0);
+        }
     }
 
     @Test
     public void testConnection() throws Exception {
-        String url = "jdbc:splice://localhost:1527/splicedb;create=true;user=splice;password=admin;useSpark=true";
-        Connection connection = DriverManager.getConnection(url, new Properties());
-        connection.setSchema(CLASS_NAME.toUpperCase());
-        Statement s = connection.createStatement();
-        ResultSet rs = s.executeQuery("select testConnection() from test");
-        String result = rs.next() ? rs.getString(1) : null;
-        Assert.assertNotNull(result);
-        Assert.assertTrue(result, result.compareTo("Got an internal connection")==0);
+        try(SpliceWatcher watcher2 = new SpliceWatcher(CLASS_NAME))
+        {
+            watcher2.setConnection( watcher2.connectionBuilder().useOLAP(true).build() );
+            ResultSet rs = watcher2.executeQuery("select testConnection() from test");
```

Comment:
missing try-with-resources for this ResultSet

Reply:
done